packages/scss/src/components/_button.scss

```diff
@@ -297,15 +296,56 @@
 		position: relative;
 
 		&:first-child {
-			border-radius: $radius 0 0 $radius;
+			border-radius: _component("button.border.radius") 0 0 _component("button.border.radius");
 		}
 
 		&:last-child {
-			border-radius: 0 $radius $radius 0;
+			border-radius: 0 _component("button.border.radius") _component("button.border.radius") 0;
 		}
 
 		&:hover, &:focus {
 			z-index: 1;
 		}
 	}
 }
+
+.button-group-item {
+
+	.button {
+		&:first-child, &:last-child {
+			border-radius: 0;
+		}
+	}
+
+	&:first-child {
+		.button {
+			border-radius: _component("button.border.radius") 0 0 _component("button.border.radius");
+		}
+	}
+
+	&:last-child {
+		.button {
+			border-radius: 0 _component("button.border.radius") _component("button.border.radius") 0;
+		}
+	}
+}
+
+.button-group-radio {
```

Comment:
We could use .button-group-input and have a single class for both checkbox / radio